compiler-core/src/docs/snapshots/gleam_core__docs__tests__canconical_link.snap

```diff
@@ -0,0 +1,361 @@
+---
+source: compiler-core/src/docs/tests.rs
+expression: "compile(config, modules)"
+---
+//// app.html
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <title>app · test_canconical · v0.1.0</title>
+    <meta name="description" content=""/>
+    <meta name="theme-color" content="#ffaff3" media="(prefers-color-scheme: light)"/>
+    <meta name="theme-color" content="#33384d" media="(prefers-color-scheme: dark)"/>
+    <link rel="stylesheet" href="./css/index.css?v=GLEAM_VERSION_HERE" type="text/css"/>
+    <!-- The docs_config.js file is provided by HexDocs and shared
+         between multiple versions of the same package. -->
+    <script src="./docs_config.js"></script>
+    <link id="syntax-theme" rel="stylesheet" href="./css/atom-one-light.min.css?v=GLEAM_VERSION_HERE"/>
+    <link rel="canconical" href="./test_canconical/app.html" />
```

Comment:
Canonical URLs need to be absolute!

Reply:
How do we know the absolute URL here? Since users could host it themselves; I am not aware of where this setting is.

Reply:
It should be set via configuration when publishing to Hex. We can omit otherwise for now